code/modules/mob/living/brain/brain_item.dm

```diff
@@ -265,11 +266,35 @@
 	prev_damage = damage
 	return
 
+/obj/item/organ/brain/proc/get_mmi_brain_sprite()
+	return "mmi_brain"
+
 /obj/item/organ/brain/alien
 	name = "alien brain"
 	desc = "We barely understand the brains of terrestial animals. Who knows what we may find in the brain of such an advanced species?"
 	icon_state = "brain-x"
 
+/obj/item/organ/brain/alien/get_mmi_brain_sprite()
+	return "mmi_brain_alien"
+
+/obj/item/organ/brain/vox
+	name = "cortical stack"
+	brain_name = "cortical stack"
+	desc = "A peculiarly advanced bio-electronic device that seems to hold the memories and identity of a Vox."
+	icon_state = "cortical-stack"
+	decay_factor = 0
+
+/obj/item/organ/brain/vox/get_mmi_brain_sprite()
+	return "mmi_cortical_stack"
+
+/obj/item/organ/brain/vox/emp_act(severity)
+	to_chat(owner, span_warning("Your head hurts."))
+	switch(severity)
+		if(EMP_HEAVY)
+			owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, rand(25, 50))
+		if(EMP_LIGHT)
+			owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, rand(0, 25))
```

Comment:
Severity is not binary, it's a scale of 1-10, 10 (EMP_HEAVY) being the strongest.